stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/controller/QueryChannelsController.kt

```diff
@@ -44,7 +46,12 @@ internal class QueryChannelsController(
     private val client: ChatClient,
     private val domainImpl: ChatDomainImpl,
 ) {
-    var newChannelEventFilter: (Channel, FilterObject) -> Boolean = { _, _ -> true }
+    var newChannelEventFilter: (Channel, FilterObject) -> Boolean = { channel, filterObject ->
+        println("JcLog: Channel->${channel.type}")
+        println("JcLog: Channel->${channel.members.map(Member::getUserId)}")
+        println("JcLog: FilterObject->$filterObject")
```

Comment:
We should probably remove these logs

Reply:
Ups, yep, it was me doing some test :facepalm: